packages/casting/src/follower-cosmjs.js

```diff
@@ -102,6 +123,18 @@ export const makeCosmjsFollower = (
     crasher = null,
   } = options;
 
+  /**
+   * @param {any} err
+   */
+  const crash = err => {
+    if (crasher) {
+      E(crasher)
+        .crash(`PROOF VERIFICATION FAILURE; crashing follower`, err)
+        .catch(e => assert(false, X`crashing follower failed: ${e}`));
```

Comment:
I chose not to change this since. I suspect that Chesterton put this fence (@mfig in this metaphor 😊) here in case the appropriate mitigation is to hard-kill the container (à la `process.abort`) instead of merely throwing. I made sure that the default is to throw. I can create an issue if the dangling promises are unintended.